src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -165,15 +165,15 @@ export default () => `
 							<div class="flex flex_container_video">
 								<div class="videos_container_video">
 									<a href="https://www.youtube.com/watch?v=Orv7fptVoUA" class="video overview">
-									<img class="video_overview" id="video_overview" />
+									<img src="./../../../../sql/workbench/contrib/welcome/media/video_overview.png" class="video_overview" id="video_overview" />
```

Comment:
You should just embed the require.toUrl call into here like you do with the localized strings

Reply:
done